### ISSUE TITLE
survey-vue: Cannot export 'Window' 

### ISSUE THREAD

**Issue by Karthik-art** [surveyjs/survey-library#4200]
### Are you requesting a feature, reporting a bug or asking a question?
Bug

```
ERROR in ./node_modules/survey-vue/survey.vue.d.ts(9,10):
9:10 Cannot export 'Window'. Only local declarations can be exported from a module.
     7 | 
     8 | export { SurveyModel as Model };
  >  9 | export { Window as SurveyWindow };
       |          ^
    10 | 
    11 | export enum DragTypeOverMeEnum {
    12 |   InsideEmptyPanel = 1,

```

### Specify your

* browser:
* browser version:
* surveyjs platform (angular or react or jquery or knockout or vue): vue2
* surveyjs version: 1.9.22

**Comment by andrewtelnov** [comment]
@Karthik-art This issue has been fixed and it will be included into next minor release v1.9.23. Here is [the related issue](https://github.com/surveyjs/survey-library/issues/4178).

Thank you,
Andrew

**Comment by Karthik-art** [comment]
> @Karthik-art This issue has been fixed and it will be included into next minor release v1.9.23. Here is [the related issue](https://github.com/surveyjs/survey-library/issues/4178).
> 
> @andrewtelnov Thanks for the update.
